docs/deferred-account-flow.md

```diff
@@ -0,0 +1,65 @@
+<!---
+Copyright 2018 The Subscribe with Google Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Deferred account creation flow (not launched)
+
+This flow allows the publisher site to continue account creation process for deferred subscriptions - subscriptions that the publisher doesn't have related account information/consent on their side. These kind of subscriptions could be created on other partner platforms.
+
+This flow shows the user their existing subscription information, prompts for the user's consent, and returns user/subscription information back to the publisher's site. See [Subscriptions APIs](./core-apis.md).
+
+Typically, the publisher's site can initiate deferred account creation when it receives the entitlements data, e.g. via `setOnEntitlementsResponse` callback. Thus the possible code could look like this:
+
+```
+function onEntitlement(entitlements) {
+  if (!entitlements.enablesThis()) {
+    return;
+  }
+  // The entitlement enables this product.
+  // 1. Check if the publisher site knows this subscription.
+  if (!myKnownSubscription(entitlements)) {
+    // 2. Start deferred account creation.
```

Comment:
I think we need something here to indicate that they have no identity in particular. That is, they may have received this subscription in the pubsub, but they have no associated user account. Maybe this?...

if (!hasAssociatedUserAccount(entitlements))

Reply:
Done